src/main/java/control/FileMenuController.java

```diff
@@ -327,34 +326,6 @@ private boolean validateCameraTypeData() {
     }
     
     /**
-     * Validate the data entered by the user in the modal to add a timeline.
-     * @return true is the data is legit, false otherwise
-     */
-    private boolean validateTimelineData() {
```

Comment:
No, this remains from the old situation where camera additions and timeline additions were still two seperate things